code/controllers/subsystems/ticker.dm

```diff
@@ -486,6 +475,24 @@ var/datum/controller/subsystem/ticker/SSticker
 
 	return SETUP_OK
 
+/datum/controller/subsystem/ticker/proc/roundstart()
+	mode.post_setup()
+	//Cleanup some stuff
+	for(var/obj/effect/landmark/start/S in landmarks_list)
+		//Deleting Startpoints but we need the ai point to AI-ize people later
+		if (S.name != "AI")
+			qdel(S)
+	// update join icon for lobbysitters
+	for(var/mob/abstract/new_player/NP in player_list)
+		if(!NP.client)
+			continue
+		var/obj/screen/new_player/selection/join_game/JG = locate() in NP.client.screen
+		JG.update_icon(NP)
```

Comment:
I wonder if it could happen that the button would not be found.

Reply:
All `/mob/abstract/new_player`s are necessarily on the lobby screen, I think.